src/pudl/output/ferc1.py

```diff
@@ -1630,8 +1653,10 @@ def single_valued_weights(cls, v: pd.DataFrame, values) -> pd.DataFrame:
             # That we are dealing with explicitly in building the trees below. Maybe it
             # should be happening here instead?
             logger.warning(
-                "Calculation forest nodes specified with conflicting weights:\n"
-                f"{v.loc[multi_valued_weights]}"
+                "Calculation forest nodes specified with conflicting weights."
+            )
+            logger.debug(
+                f"Nodes with conflicting weights:\n{v.loc[multi_valued_weights]}"
             )
```

Comment:
this was just reaaaly noisy.

Reply:
I thought I shut this guy up on my branch as well...